tests/test_trim_util.py

```diff
@@ -290,3 +299,34 @@ def test_log_last_triggered_exc_logged(self, mock_log_exc, mock_time):
         trim_util._log_last_triggered(session, 'sr_uuid')
 
         self.assertEqual(1, mock_log_exc.call_count)
+
+    @mock.patch("trim_util.lvutil.LVM_SIZE_INCREMENT", EMPTY_VG_SPACE)
+    @mock.patch('trim_util.lvutil')
+    @mock.patch('lock.Lock')
+    @mock.patch('util.sr_get_capability')
+    @testlib.with_context
+    def test_do_trim_creates_an_lv_onfilledSR_not_supported(self,
```

Comment:
The name of the test case is rather convoluted. Can we have something simple like "test_do_trim_returns_exception_when_sr_full"